vm-setup/roles/v1aX_integration_test/tasks/deprovision_controlplane.yml

```diff
@@ -1,33 +1,7 @@
 ---
-  - name: Deprovision Ubuntu based controlplane node
-    block:
-      - name: Copy generated controlplane_ubuntu.yaml
-        copy:
-          src: "{{ TEMP_GEN_DIR }}/{{ CAPI_VERSION }}_controlplane_ubuntu.yaml"
-          dest: "/tmp/controlplane_ubuntu.yaml"
-
-      - name: Deprovision Ubuntu master node.
-        k8s:
-          state: absent
-          src: "/tmp/controlplane_ubuntu.yaml"
-          namespace: "{{ NAMESPACE }}"
-        ignore_errors: yes
-        register: deprovision_ubuntu_master_node
-    when: IMAGE_OS == "Ubuntu"
-
-  - name: Deprovision CentOS based controlplane node
-    block:
-      - name: Copy generated controlplane_centos.yaml
-        copy:
-          src: "{{ TEMP_GEN_DIR }}/{{ CAPI_VERSION }}_controlplane_centos.yaml"
-          dest: "/tmp/controlplane_centos.yaml"
-
-      - name: Deprovision CentOS master node.
-        k8s:
-          state: absent
-          src: "/tmp/controlplane_centos.yaml"
-          namespace: "{{ NAMESPACE }}"
-        ignore_errors: yes
-        register: deprovision_centos_master_node
-    when: (IMAGE_OS == "Centos") or
-          (IMAGE_OS == "")
+  - name: Deprovision Ubuntu master node.
```

Comment:
The name of the task is misleading as the same task used for both Ubuntu & Centos.